themes/vocabulary_theme/templates/page-with-toc.html

```diff
@@ -17,7 +17,7 @@ <h1>{{ this.title }}</h1>
         <div class="column is-one-quarter">
           <div class="row">
             {% if this.is_child_of('/contributing-code') %}
-              <aside class="menu sidebar-menu">
+              <aside class="menu sidebar-menu" role="complementary" aria-label="Contributing Code Sidebar">
```

Comment:
`aside` already has a semantic role, unsure this is necessary. Additionally if it needs a label it should likely have a heading within it to support that.